src/CoreWCF.Http/tests/SimpleWSFedBinding.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+using CoreWCF;
+using CoreWCF.Configuration;
+using CoreWCF.IdentityModel.Configuration;
+using CoreWCF.IdentityModel.Tokens;
+using CoreWCF.Security;
+using Helpers;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.IdentityModel.Tokens.Saml;
+using Microsoft.IdentityModel.Tokens.Saml2;
+using Xunit;
+using Xunit.Abstractions;
+
+
+namespace WSFed
+{
+    public class SimpleWSFedBinding
+    {
+        private readonly ITestOutputHelper _output;
+
+        public SimpleWSFedBinding(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        
+       //[Theory]
+       // [InlineData(Saml2Constants.OasisWssSaml2TokenProfile11,false)]
```

Comment:
I would rather leave the test attributes uncommented and use the Skip = true property to disable the tests.